docs/source/model-registry.rst

```diff
@@ -78,47 +79,170 @@ version to one of the other valid stages.
 
 .. code-block:: py
 
-   mlflow.register_model("runs:/e148278107274..832/artifacts/model", "EmailMarketingCampaign")
+    result=mlflow.register_model(“runs:/d16076a3ec534311817565e6527539c0/artifacts/sk-model”,
+                    ”sk-learn-random-forest-reg”)
+
+
+As with above ``mlflow.sklearn.log_model(...)``, this method creates version 1 of the specified model and it returns a single `ModelVersion <https://www.mlflow.org/docs/latest/python_api/mlflow.entities.html#mlflow.entities.model_registry.ModelVersion>`_ MLflow object.
 
-If the model has not been logged, pass ``registered_model_name`` to ``log_model`` to log the model under the specified registered model.
+And finally, you can use the `MLflow Client Tracking API <https://www.mlflow.org/docs/latest/python_api/mlflow.tracking.html#mlflow.tracking.MlflowClient.create_registered_model>`_ to create a new registered model. If the model name exists, this will throw an ``MLflowException`` since creating a new registered model requires a unique name.
 
 .. code-block:: py
 
-   mlflow.tensorflow.log_model(tf_saved_model_dir=saved_estimator_path,
-                               tf_meta_graph_tags=[tag_constants.SERVING],
-                               tf_signature_def_key="predict",
-                               artifact_path="model",
-                               registered_model_name="EmailMarketingCampaign")
+   client = MlflowClient()
+   client.create_registered_model("sk-learn-random-forest-reg-model")
 
-.. note:: If the registered model does not already exist, the registry will create one before creating a new model version.
+While the method above creates an empty registered model with no version associated, the method below creates a new version of the model.
 
-To confirm that the model has successfully been registered, call the ``get_model_version_details`` API:
+.. code-block:: py
+
+    client = MlflowClient()
+    result = client.create_model_version(name="sk-learn-random-forest-reg-model",
+                source = "mlruns/0/d16076a3ec534311817565e6527539c0/artifacts/sk-model",
+                run_id = "d16076a3ec534311817565e6527539c0")
+
+In contrast, ``mlflow.register_model()`` and ``mlflow.model_flavor.log_model()`` will create a new version in the registry.
+
+Adding or Updating Model Descriptions
+-------------------------------------
+
+At any point in a model’s lifecycle development, you can update a model version's description using the MLflow Tracking API.
 
 .. code-block:: py
 
-   client = mlflow.tracking.MlflowClient()
-   client.get_model_version_details(name = "EmailMarketingCampaign",
-                                    version = 12)
+    client = MlflowClient()
+    client.update_model_version(name=”sk-learn-random-forest-reg-model”,
+            version=1,
+            description="This model version is a random forest containing 100 decision trees that was trained in scikit-learn.")
```

Comment:
Since the code snippet does not wrap, maybe make the description a bit shorter.

Reply:
Wil make is shorter